responses.py

```diff
@@ -692,6 +692,23 @@ def stop(self, allow_assert=True):
                 )
             )
 
+    def assert_call_count(self, url, count):
+        call_count = len(
+            [
+                1
+                for call in self.calls
+                if call.request.url == _ensure_url_default_path(url)
```

Comment:
Should we fail early when the request is empty?

Reply:
I don't know how I can test it `when the request is empty`.(TBH I didn't understand it).

FYI, we can use this for testing 0 calls.
https://github.com/getsentry/responses/pull/318/files#diff-a95e1a562f3879ddb72fe81003f48c52R1164

Please explain the case, I will change the patch.
Thanks